### PR TITLE
Allow FAQs to be shown on home/sponsor/registration pages

### PR DESCRIPTION
**Pull request by epsy**
https://trello.com/c/LMRXedBp/133-implement-sections-of-faq-on-sponsors-and-registration-page

### CODE REVIEW

**Comment by epsy** [comment]
screenshot 2017-05-24 at 22 55 09

File: pybay/urls.py [hunk 1]
```diff
@@ -12,23 +12,23 @@
 from symposion.sponsorship import urls as sponsor_urls
 from symposion.speakers import urls as speaker_urls
 
-from pybay.views import (
-    pybay_cfp_create, pybay_sponsors_list,
-    pybay_faq_index, pybay_speakers_list)
+from pybay import views
```
Comment (mzdaniel) [inline pybay/urls.py]: This change seems more a matter of taste than what the PR is asking for and adds an extra namespace that is not needed. Lets not mix the two.

**Comment by mzdaniel** [comment]
Except for the comment, this looks like a fantastic job Yann!  The PR is in staging now for UX review.

File: pybay/urls.py (hunk 1 above)
Comment (epsy) [reply pybay/urls.py]: Done

File: pybay/urls.py [hunk 2]
```diff
@@ -13,21 +13,26 @@
...
 
 WIKI_SLUG = r"(([\w-]{2,})(/[\w-]{2,})*)"
 
 
+faq_view = FaqTemplateView.as_view
+
+
 urlpatterns = [
-    url(r"^$", TemplateView.as_view(template_name="frontend/index.html"), name="home"),
+    url(r"^$", faq_view(template_name="frontend/index.html", faq_filter="show_on_home"), name="home"),
```
Comment (PirosB3) [inline pybay/urls.py:29]: Why are you calling this `faq_view`? isn't it home page?

Comment (epsy) [reply pybay/urls.py:29]: It uses data from the FAQs. If the name is too confusing I could change it to something else

**Review by PirosB3 — APPROVED** [review]

**Comment by PirosB3** [comment]
@mzdaniel merged when you are happy with this

**Comment by grace-law** [comment]
Yann, this is great stuff.  I anticipate this feature will save me and other devs over 20 hours a week getting content on the website and editing.

Minor changes:  
1) Can you please move the "Frequently Asked Questions" text to center and make that Black (over-ride current CSS rules) for home, tickets and sponsors prospectus page.
2) Can you please change the questions  to the darker Blue and Bold (same size)

**Comment by mzdaniel** [comment]
I am merging this code now. Great work Yann!

**Pull request merged**